display_information/update-graphics/src/main/java/com/esri/samples/update_graphics/UpdateGraphicsSample.java

```diff
@@ -51,17 +53,14 @@
 
 public class UpdateGraphicsSample extends Application {
 
-  private boolean isUpdateLocationActive;
   private List<SimpleMarkerSymbol> markers;
-  private Button updateLocationButton;
   private Button updateDescriptionButton;
   private ComboBox<String> symbolBox;
 
-  private ArcGISMap map; // keep loadable in scope to avoid garbage collection
   private MapView mapView;
-  private Graphic selectedGraphic;
+  private Graphic identifiedGraphic = null;
```

Comment:
we don't have to assert it's null here, the compiler will auto set all member variables to a reasonable default, generally null in this case.
```suggestion
  private Graphic identifiedGraphic;
```